ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -146,6 +146,47 @@ def var_mean(input, dim, unbiased, keepdim=False, *, out=None):
     temp_mean = ivy.mean(input, axis=dim, keepdims=keepdim, out=out)
     return (temp_var, temp_mean)
 
+@to_ivy_arrays_and_back
+# sorted: if true -> ascending order
+def unique(input, sorted=True, return_inverse=False, return_counts=False, dim=None):
+   var_type = ivy.dtype(input)
+   print("---------------------------New Test----------------------")
+   print("input: ",input )
+   print("dim: ", dim)
+
+   input_shape = input.shape
+   input = input.reshape(-1, input_shape[-1])

```

Comment:
this should be tuple else it will give an exception, reshape((-1, input_shape[-1]))

Reply:
> this should be tuple else it will give an exception, reshape((-1, input_shape[-1]))

Alright! Just one quick question, do you mean that what you coded (the green square) should be replaced for what I was doing for "dim" (what is on the red square)? 

<img width="1280" alt="Sin título" src="https://user-images.githubusercontent.com/58478529/223214119-12829ff4-5f52-452e-8c9a-1e65bc93d639.png">

And regarding the falsifying example, sure, here you have the received arguments:

- input =  ivy.array([-1., -1.])
- sorted = False 
- return_inverse=True
- return_counts=False
- dim = 0

-Expected output by the ground truth backend for return_inverse : array([0, 0])
-Returned output by the ivy.unique_inverse() function: array([-1], dtype=bfloat16),

I am also attaching this if it is helpful: "You can reproduce this example by temporarily adding @reproduce_failure('6.55.0', b'AXicY2BkAAMoBaaR2DAmAACpAAc=') as a decorator on your test case"

By the way. Just to leave it clear, I had this discussion regarding the "sorted" argument in Discord #ivy-frontend-tests https://discord.com/channels/799879767196958751/1028267758028337193/1079809888873627648 in which we agreed that there was no need to sort it in unique() , just in case you would like to take a look.

I'll work on the rest (the counts flag) in the meantime thank you so much for your guidance. Have an excellent day and I'm right here if you need anything :) @zaeemansari70